android/app/build.gradle

```diff
@@ -28,6 +28,9 @@ android {
         targetSdk = flutter.targetSdkVersion
         versionCode = flutter.versionCode
         versionName = flutter.versionName
+        manifestPlaceholders += [
+                'appAuthRedirectScheme': 'com.example.grueneapp://oauthredirect'
```

Comment:
:question: Is this something that is relevant for some other place? If yes, we should probably already adjust the app id to the correct string.
@maxpfe is there a requirement for a specific app id? Or should we just choose something of our choice?

Reply:
I'm not sure yet where this is relevant. Ii think we need to change it some day. But I don't know what it should be named. So let's keep it for now and create a new issue when we know what the app id should be.